scripts/completion.sh

```diff
@@ -0,0 +1,72 @@
+# bash completion support for xkbcli.
+
+___xkbcli_main()
+{
+    _init_completion -s || return
+
+    local i=1 cmd
+
+    # Find subcommand
+    while [[ "$i" -lt "$COMP_CWORD" ]]
+    do
+        local s="${COMP_WORDS[i]}"
+        case "$s" in
+            -*) ;;
+            *)
+            cmd="$s"
+            break
+            ;;
+        esac
+        (( i++ ))
+    done
+
+    local subcommands=(
```

Comment:
It would be good to avoid duplicating this list here, since it at risk of getting out of sync. Possible alternatives (in my order of preference) are to parse the `--help` output, add a hidden `xkbcli` subcommand/flag which lists the subcommands and use that here, or fill using a template.